sql/src/main/java/org/apache/druid/sql/calcite/schema/SystemSchema.java

```diff
@@ -92,6 +93,9 @@
   private static final String SERVER_SEGMENTS_TABLE = "server_segments";
   private static final String TASKS_TABLE = "tasks";
 
+  private static final long AVAILABLE_IS_OVERSHADOWED_VALUE = 0L;
```

Comment:
hm, if we use these, I think `IS_OVERSHADOWED_FALSE` would be clearer, and it should have `IS_OVERSHADOWED_TRUE` as well

Reply:
ok, yeah, i was having hard time coming up with good constant names for these.